examples/mnist/train.cpp

```diff
@@ -79,10 +79,12 @@ static void train_lenet(const std::string& data_dir_path) {
 
   progress_display disp(static_cast<unsigned long>(train_images.size()));
   timer t;
-  int minibatch_size = 10;
+  int minibatch_size = 16;
   int num_epochs     = 30;
 
-  optimizer.alpha *= static_cast<tiny_dnn::float_t>(std::sqrt(minibatch_size));
+  optimizer.alpha *=
+    std::min(tiny_dnn::float_t(4),
+             static_cast<tiny_dnn::float_t>(std::sqrt(minibatch_size)));
```

Comment:
@Randl Thank you for the answer! :bow: I haven't studied about neural networks so there're a lot to learn. 

@pliptor Than you for the detailed explanation. :joy: It's pretty helpful. I'm not sure if maximum of alpha should be adjusted per network. Talking about the casting, I need to check its consequences.